### PR TITLE
fix: write default app config on app init

### LINKED ISSUE #355: aio app init -y doesn't create the .aio file
Running `aio app init -y`  doesn't create the `.aio` file

### PR DESCRIPTION
**Pull request by sandeep-paliwal**
Always create .aio file with default app config for app init command.
Fixes #355 

## Description

When aio app is initialised with -y flag, .aio file was not created. 
Fixed this by writing .aio file with default app config at app init. When import or CLI login is used, .aio file will be merged with project config.
Default app config includes, name of actions, dist and web-src folder. This will help users to customise app if required.

## How Has This Been Tested?

unit tests, manual app init test

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/lib/import.js [hunk 1]
```diff
@@ -95,6 +95,28 @@ function loadAndValidateConfigFile (fileOrBuffer) {
...
+ * @param {string} parentDir the parent folder to write the .aio file to
+ * @param {object} [flags] flags for file writing
+ * @param {boolean} [flags.overwrite=false] set to true to overwrite the existing .env file
+ * @param {boolean} [flags.merge=false] set to true to merge in the existing .env file (takes precedence over overwrite)
+ * @param {boolean} [flags.interactive=false] set to true to prompt the user for file overwrite
+ * @returns {Promise} promise from writeFile call
+ */
+function writeDefaultAppConfig (parentDir, flags) {
+  // write app config to .aio file
+  const appConfig = {
```
Comment (shazron) [inline src/lib/import.js:110]: as an output, this has to be tested that it doesn't change unexpectedly

**Review by shazron — CHANGES_REQUESTED** [review]

File: src/lib/import.js (hunk 1 above)
Comment (sandeep-paliwal) [reply src/lib/import.js:110]: @shazron there is test for `writeDefaultAppConfig ` in import.test.js to check output

Comment (shazron) [reply src/lib/import.js:110]: Saw the test. I was thinking more like the contents here:
```
app: {
       actions: 'actions',
       dist: 'dist',
       web: 'web-src'
     }
```
A user might be relying on those values we output, so we shouldn't delete or change them (without a major version release) and a test would ensure this contract

Comment (sandeep-paliwal) [reply src/lib/import.js:110]: Added the check to verify default aio content.

**Review by shazron — APPROVED** [review]

**Pull request merged**